pkgs/applications/networking/instant-messengers/utox/default.nix

```diff
@@ -1,22 +1,22 @@
 { stdenv, fetchFromGitHub, cmake, pkgconfig, libtoxcore, filter-audio, dbus, libvpx, libX11, openal, freetype, libv4l
-, libXrender, fontconfig, libXext, libXft, utillinux, git, libsodium }:
+, libXrender, fontconfig, libXext, libXft, utillinux, git, libsodium, libopus, check }:
 
 stdenv.mkDerivation rec {
   name = "utox-${version}";
-  # >= 0.14 should have unit tests and dbus support
-  version = "0.13.1";
+
+  version = "0.16.1";
 
   src = fetchFromGitHub {
     owner  = "uTox";
     repo   = "uTox";
     rev    = "v${version}";
-    sha256 = "07aa92isknxf7531jr9kgk89wl5rvv34jrvir043fs9xvim5zq99";
+    sha256 = "0ak10925v67yaga2pw9yzp0xkb5j1181srfjdyqpd29v8mi9j828";
   };
 
   buildInputs = [
     libtoxcore dbus libvpx libX11 openal freetype
     libv4l libXrender fontconfig libXext libXft filter-audio
-    libsodium
+    libsodium libopus check
```

Comment:
I am assuming that ```check``` is for the ```checkPhase```. Can we add it to ```checkInputs``` instead then? Alternatively, ```nativeBuildInputs```.

Reply:
I think `checkInputs` only exists in `buildPythonPackage`.

Reply:
I think you're right @Mic92  - always nice to learn something new.

In that case it should be ```nativeBuildInputs```.

Reply:
Good point. I've moved check to nativeBuildInputs, and added and appropriate "checkTarget" configuration.